includes/admin_import.php

```diff
@@ -733,7 +733,7 @@ function tmsl_import_page() {
 							<?php if ($tsml_sharing == 'restricted') {?>
 								<details>
 									<summary><strong><?php _e('Authorized Apps', '12-step-meeting-list')?></strong></summary>
-									<p><?php _e('You may allow access to your meeting data for specific purposes, such as the <a target="_blank" href="https://meetingguide.org/">Meeting Guide App</a>.')?>
+									<p><?php _e('You may allow access to your meeting data for specific purposes, such as the <a target="_blank" href="https://meetingguide.org/">Meeting Guide App</a>.')?></p>
```

Comment:
Also on this one

Reply:
Should it also be on this line?
`<input type="text" name="tsml_add_sharing_key" placeholder="<?php _e('Meeting Guide')?>">`

Reply:
Should it also be on this line?
`<input type="text" name="tsml_add_sharing_key" placeholder="<?php _e('Meeting Guide')?>">`

Reply:
Yes, good catch!